custom/icds_reports/static/js/angular-services/locations.service.js

```diff
@@ -1,6 +1,17 @@
 window.angular.module('icdsApp').factory('locationsService', ['$http', '$location', function($http, $location) {
     var url = hqImport('hqwebapp/js/initial_page_data').reverse;
     var gtag = hqImport('analytix/js/google').track;
+
+    function transformLocationTypeName(locationTypeName) {
+        if (locationTypeName === 'awc') {
+            return locationTypeName.toUpperCase();
```

Comment:
Any reason to use this rather than a constant/literal

Reply:
Not really, other than it was just moved from somewhere else.